torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -57,7 +57,11 @@ struct RecvWork {
   torch::Tensor payload_;
 };
 
+#ifdef _WIN32
+class ProcessGroupAgent : public RpcAgent {
+#else
 class TORCH_API ProcessGroupAgent : public RpcAgent {
+#endif
```

Comment:
Would that solve the compilation problem?
```suggestion
class C10_EXPORT ProcessGroupAgent : public RpcAgent {
```